spark-2.4/pom.xml

```
@@ -4,11 +4,11 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>neo4j-connector-apache-spark_${scala.binary.version}_2.4</artifactId>
+    <artifactId>neo4j-connector-apache-spark_${scala.binary.version}</artifactId>
     <name>neo4j-connector-apache-spark-2.4</name>
     <description>Spark 2.4 for Neo4j Connector for Apache Spark using the binary Bolt Driver</description>
 
-    <version>4.0.0</version>
+    <version>4.0.1_for_spark_2.4</version>
```

```suggestion
    <version>${parent.version}_for_spark_2.4</version>
```

also please update the two dependencies below in the pom as this:

```
   <dependencies>
        <dependency>
            <groupId>neo4j-contrib</groupId>
            <artifactId>neo4j-connector-apache-spark_${scala.binary.version}_common</artifactId>
            <version>${parent.version}</version>
        </dependency>
        <dependency>
            <groupId>neo4j-contrib</groupId>
            <artifactId>neo4j-connector-apache-spark_${scala.binary.version}_test-support</artifactId>
            <version>${parent.version}</version>
            <scope>test</scope>
        </dependency>
    </dependencies>
```

and do the same for `spark 3`